sycl/include/CL/sycl/info/info_desc.hpp

```diff
@@ -334,6 +336,8 @@ PARAM_TRAITS_SPEC(device, reference_count, cl_uint)
 PARAM_TRAITS_SPEC(device, max_num_sub_groups, cl_uint)
 PARAM_TRAITS_SPEC(device, sub_group_independent_forward_progress, bool)
 PARAM_TRAITS_SPEC(device, sub_group_sizes, vector_class<size_t>)
+PARAM_TRAITS_SPEC(device, kernel_kernel_pipe_support, bool)
```

Comment:
Are these extensions aligned with the spec?

Reply:
Are these new traits documented anywhere?

Reply:
Yes, https://github.com/intel/llvm/blob/78f1b51e63ad051cb82f3fa57687bc8cfadc2335/sycl/doc/extensions/DataFlowPipes/data_flow_pipes.asciidoc device query section.

Reply:
What about `spirv_version`?

Reply:
You caught me. There is not such query in SYCL and that's actually strange, because even OpenCL has. I'll talk to the spec editors then.

Reply:
Removed query for SPIR-V 1.1.